src/ima.js

```diff
@@ -1466,6 +1466,10 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
   _playAdByConfig(): boolean {
     return !!(this.config.adTagUrl || this.config.adsResponse);
   }
+
+  _isVideoAd(): boolean {
+    return this._currentAd && this._currentAd.getContentType().startsWith('video');
```

Comment:
what are the possible options for content type? is it enum? seems a bit problematic to rely on 3rd party strings.

Reply:
The content type is the standard MIME type of the file. I don't see another way to distinguish linear image ads vs linear video ads.
The only caveat is that the field [can be an empty string](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/reference/js/google.ima.Ad#getContentType if content type is unavailable but I don't know how common that scenario is.

Reply:
did you check with IMA team? open a question in IMA forums https://groups.google.com/g/ima-sdk

Reply:
It takes them a while to respond but sure, I can ask

Reply:
better then adding a regression we cannot handle for all video ads, what is current issue without this fix? any risk for video ads?

Reply:
This PR and [the original 2 PRs](https://github.com/kaltura/playkit-js-ima/pull/209) handle 
1. a wrong order of events in linear image ads
2. linear image ads having UI issues because they weren't identified as image ads

The first issue doesn't happen in video ads, the second issue is exactly the content type issue which we're trying to clarify here.

Reply:
SDK team's answer: contentType will be video/... or image/..., and content type won't be empty in a playing ad https://groups.google.com/g/ima-sdk/c/aUZYp-rch5A

Reply:
Could be text/... too but not in a video ad